examples/network/custom_region_demo.py

```diff
@@ -90,8 +90,11 @@ def createNetwork(dataSource):
   # In this demo, we have modified it using sys.path.append since we need it to
   # have an effect on this program.
   sys.path.append(os.path.dirname(os.path.abspath(__file__)))
-  # Add custom region package to network
-  Network.registerRegionPackage("custom_region")
+  
+  custom_region = __import__("custom_region", fromlist=["IdentityRegion"])
```

Comment:
How about we change the filename to be different than the class name to make sure this works? And use a regular import statement.

```
# In import section
from custom_region import identity_region

Network.registionRegion(identity_region.IdentityRegion)
```